std_msgs/Header header
bool orientation_as_quat
Pose[] pose_array
builtin_interfaces/Time[] t_k #desired execution of the trajectory
